lib/vrt/version.rb

```diff
@@ -1,3 +1,3 @@
 module Vrt
-  VERSION = '0.3.2'.freeze
+  VERSION = '0.4.0'.freeze
```

Comment:
Can this be a patch update?  Seems like nothing is breaking since the mapping was broken before and now works.

Reply:
I bumped minor version because we added a whole new vrt version. Seemed a sufficient enough change. But if you really think it should be patch...

Reply:
I think since the vrt version doesn't actually contain any changes other than the bug fix we are ok to just do a patch?

@barnett thoughts?

I guess I'm not strongly pulled either way.

Reply:
Ya I agree with @tessereth here. Even though the contents of that version is a patch in the VRT, we have always updated the VRT gem with a minor version change upon a new version of the VRT being published. I don't think it's the responsibility of this gem to examine the contents of the VRT submodule but just minor update regardless.